openssl/src/ssl/connector.rs

```diff
@@ -414,10 +414,10 @@ cfg_if! {
                 GeneralName, X509NameRef, X509Ref, X509StoreContext, X509StoreContextRef,
                 X509VerifyResult,
             };
+            use once_cell::sync::Lazy;
 
-            lazy_static! {
-                pub static ref HOSTNAME_IDX: Index<Ssl, String> = Ssl::new_ex_index().unwrap();
-            }
+            pub static HOSTNAME_IDX: Lazy<Index<Ssl, String>> =
+                Lazy::new(|| Ssl::new_ex_index().unwrap());
```

Comment:
If use use a `OnceCell` instead you can use `get_or_try_init` to avoid the unwrap here: https://docs.rs/once_cell/1.5.2/once_cell/sync/struct.OnceCell.html#method.get_or_try_init